src/lessons/01_getting_started/getting-started.slides.html

```diff
@@ -0,0 +1,549 @@
+
+              <preview-window window-title="Movie Finder"
+                              icon-class="fa-globe-asia"
+                              address="https://navya-shajan.github.io/movieApp/"
+                              class="r-stretch fragment">
+                <iframe data-src="https://navya-shajan.github.io/movieApp/"></iframe>
+              </preview-window>
+            </div>
+          </section>
+        </section>
+        
+        <section>
+          <section data-auto-animate data-timing="10">
+            <h2 class="r-fit-text">Setting up the development environment</h2>
+          </section>
+          
+          <section data-auto-animate data-timing="60">
+            <h2 style="font-size: 1em">Setting up the development environment</h2>
+            
+            <div class="columns align-start justify-center">
+              <div>
+                <img data-src="images/vscode.png" alt="VS Code">
+                <p style="font-size: .6em">Visual Studio Code</p>
+              </div>
+              <div>
+                <img data-src="images/github.png" alt="Github">
+                <p style="font-size: .6em">Git & Github</p>
+              </div>
+            </div>
+          </section>
+          
+          <section data-auto-animate>
+            <h3>Visual Studio Code</h3>
+            <ul>
+              <li>Download Visual Studio Code from <a href="https://code.visualstudio.com/Download">code.visualstudio.com/download</a></li>
+              <li>Run the installer, follow its instructions</li>
+              <li>Open VS Code, and from there open or create a directory which serves as your workspace</li>
+            </ul>
+          </section>
+          
+          <section data-auto-animate>
+            <h3>Visual Studio Code</h3>
+            <img data-src="images/vscode-open-folder.png" alt="Opening a new folder in VS Code" class="r-stretch">
+          </section>
+          
+          <section data-auto-animate>
+            <h3>Visual Studio Code</h3>
+            
+            <p>For now we're done. We'll come back to Visual Studio Code at a later point toady.</p>
+          </section>
+        </section>
+        
+        <section>
+          <section data-auto-animate data-timing="10">
+            <h2>Git and Github</h2>
+          </section>
+          
+          <section data-auto-animate data-timing="30">
+            <h2>Git and Github</h2>
+            <p>What's the difference?</p>
+          </section>
+          
+          <section data-auto-animate data-timing="30">
+            <h2>Git and Github</h2>
+            
+            <div class="git-github-comparison">
+              <div>
+                <div class="git" title="Git"><div>Git</div></div>
+                <div class="is-to">is to</div>
+                <div class="github" title="Github"><div>Github</div></div>
+                
+                <div class="what fragment" data-fragment-index="0">what</div>
```

Comment:
what do you think about changing into `whereas` or even `while` instead?

```suggestion
                <div class="what fragment" data-fragment-index="0">while</div>
```

Reply:
I don't think that works logically 😅 I'm making a metaphor here. "while" or "whereas" indicates that there are differences though, doesn't it?